src/octree/types.rs

```diff
@@ -8,7 +8,7 @@ use serde::{Deserialize, Serialize};
 pub(crate) enum NodeContent<T: Clone, const DIM: usize = 1> {
     #[default]
     Nothing,
-    Internal(u32), // cache data to store the enclosed nodes
```

Comment:
This change should not be here as well, but after the merge I needed to fix some errors manually